service/github/github.go

```diff
@@ -309,13 +309,21 @@ func buildSingleSuggestion(c *reviewdog.Comment, s *rdf.Suggestion) (string, err
 	if start.GetColumn() > 0 || end.GetColumn() > 0 {
 		return buildNonLineBasedSuggestion(c, s)
 	}
+
+	// A test suggestion that uses four backticks w/o code fence block.
```

Comment:
[nit]
How about moving this comment to the util package and add a comment to the package that the util supports GitHub Flavor Markdown? 

I also  tested GitLab makrdown and I confirmed it supported an arbitrary number of backticks, so you can also note that the util supports the GitLab.